Python/ogusa/macro_output.py

```diff
@@ -55,9 +55,9 @@ def dump_diff_output(baseline_dir, policy_dir):
     
     ## Load SS results
     ss_policy_path = os.path.join(policy_dir, "SSinit", "ss_init_vars.pkl")
-    ss_policy = pickle.load(open( tpi_macro_vars_policy_path, "rb" ))
+    ss_policy = pickle.load(open( ss_policy_path, "rb" ))
```

Comment:
this change looks correct, because the comment says "Load SS results". Does that mean it was loading the incorrect file before?